packages/api-server/README.md

```diff
@@ -0,0 +1,29 @@
+Redwood [Fastify](https://www.fastify.io) Server
+
+## package.json Server Binaries
+
+### `rw-serve`
+Not intended for production use.
+- Runs web on redwood.toml web.port (default 8910)
+- API listens on web port at path redwood.toml web.apiUrl
+- Command Options:
+    - port (default 8910)
+    - socket (optional)
+    - apiHost (default redwood.toml web.apiUrl)
+
+### `rw-server api` or `rw-api-server`
```

Comment:
```suggestion
### `rw-serve api` or `rw-api-server`
```

Reply:
cc @dac09 @jtoar @dthyresson 
Naming things is hard. Here's why I chose "server":
- it is consistent with the existing bin `rw-api-server`
- it avoids being mistaken for `yarn rw serve`
- my mental model for this PR is about programatically accessing the Fastify server, which wasn't just about `serve`

I'll defer to the group vote.

Reply:
> * it avoids being mistaken for `yarn rw serve`

Totally caught me here! 

But now I'm confused. So why is it `rw-server api` and without the 'r' in `rw-serve web`? (The header on line 22)

Reply:
😂 Cause I botched it.

Updated in the latest commit here: https://github.com/redwoodjs/redwood/pull/4278/commits/146f9b3f688aeafd4370226a984db819e6e3d015 

Resolving this discussion for now but please re-comment/open if needed.